packages/api-explorer/src/components/DocSdkUsage/DocSdkUsage.tsx

```diff
@@ -40,31 +48,59 @@ interface DocSdkUsageProps {
  *  links to the source files
  */
 export const DocSdkUsage: FC<DocSdkUsageProps> = ({ method }) => {
-  const tabs = useTabs()
   const { examples } = useContext(LodeContext)
   const languages = findExampleLanguages(examples, method.name)
   if (languages.length === 0) return <></>
 
+  const tableExamples = prepareExampleTableData(
+    languages,
+    examples,
+    method.operationId
+  )
+
   return (
     <CollapserCard id="examples" heading="SDK Examples">
-      <>
-        <TabList {...tabs}>
-          {languages.map((language) => (
-            <Tab key={language}>{language}</Tab>
-          ))}
-        </TabList>
-        <TabPanels {...tabs} pt="0">
-          {languages.map((language) => (
-            <TabPanel key={language}>
-              <DocExamples
-                lode={examples}
-                language={language}
-                operationId={method.operationId}
-              />
-            </TabPanel>
-          ))}
-        </TabPanels>
-      </>
+      <Card height="auto" p="medium" mt="small">
+        <Flex alignItems="center" mx="medium" minHeight="2rem">
+          <FlexItem flexBasis="50%">
+            <Text fontWeight="semiBold">Filename</Text>
+          </FlexItem>
+          <FlexItem flexBasis="40%">
+            <Text fontWeight="semiBold">Language</Text>
+          </FlexItem>
+          <FlexItem flexBasis="10%">
+            <Text fontWeight="semiBold" style={{ float: 'right' }}>
+              Line
+            </Text>
+          </FlexItem>
+        </Flex>
+        {tableExamples.map((exampleRow: ExampleLinkTableRow, i: number) => (
+          <div key={i}>
+            <Divider appearance="light" />
+            <Flex alignItems="center" mx="medium" minHeight="2rem">
+              <FlexItem flexBasis="2%">
+                <Icon
+                  style={{ float: 'right' }}
+                  mr="small"
+                  name="IdeFileGeneric" // TODO use InsertDriveFile when available
```

Comment:
Comment in the middle of JSX?

Reply:
I was surprised Prettier let me do it... Will resolve in the merge commit